Dfe.Academies.External.Web/Pages/DeleteApplication.cshtml.cs

```diff
@@ -0,0 +1,66 @@
+using Dfe.Academies.External.Web.Pages.Base;
+using Dfe.Academies.External.Web.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Dfe.Academies.External.Web.Pages
+{
+    public class DeleteApplicationModel : BasePageEditModel
```

Comment:
Is this inheriting from an EditModel because you are treating it as an update?

I'd remove the overrides we aren't using.

Reply:
we cant inherit from the editmodel without them..see addcontributor page for an example